packages/mobile/src/services/env.ts

```diff
@@ -1,3 +1,4 @@
 export const env = {
-  EAGER_DISCOVERY_NODES: process.env.REACT_APP_EAGER_DISCOVERY_NODES
```

Comment:
really close! for mobile, its `Config.BLAH` where BLAH is the thing minus the REACT_APP, so `ENVIRONMENT: Config.ENVIRONMENT` but also don't trip much cause this is all fleshed out in the the native-reloaded branch actually